docs/testplan/PFC_Snappi_Additional_Testcases.md

```diff
@@ -0,0 +1,65 @@
+|           11 | PFCWD                    | **PFCWD ENABLED FORWARD MODE TEST:**<br>Test case to ensure that both lossless and lossy priority traffic are forwarded when PFCWD is enabled in forward mode and egress ports faces PFC storm. DUT does not send PFCs out of ingress port and no drops for both lossy and lossless traffic.                              | - Two 100Gbps Ingress and single 100Gbps egress <br>- Repeat test with 2x400Gbps ingress and 1x400Gbps egress <br>- 1024B packet-size                                                                                                  | - 30% - Lossless Priority 3 and 4 per ingress LC <br>- 10% Lossy Background Flow [Priority 0, 1, and 2] per ingress LC. <br>- Enable PFCWD with "action" set to FORWARD <br>- Send continuous PFC for lossless Priorities for configured interval.                     | - Single LC - Multiple ASIC <br>- Single LC - Single ASIC <br>- Multiple LC      | 1. There should be no packet drops on DUT for lossy background flows. <br>2. The Priority 3 and 4 packets are forwarded to the Traffic Receiver. <br>3. After the PFC packets are stopped, the lossless PRIO traffic should continue without any drops.<br>4. No PFCs sent from DUT ingress to traffic transmitter. <br>5. PFCWD drop counters should not increment.                                                                                                                           |         
+|           12 | PFCWD (MACSEC)           | **TEST PFCWD BEHAVIOR OVER MACSEC ENABLED LINK:** <br>Aim of the test case is to ensure that PFCWD drops the lossless priority traffic on detecting PFC storm on MACSEC enabled DUT and mix of lossless and lossy traffic.                                                                                                | - Single 100Gbps Ingress and single 100Gbps egress - PFCWD(Drop) over MACSEC <br>- Repeat with Single 400Gbps ingress and Single 400Gbps egress. <br> - 1024B packet-size.                                                             | - 90% - Lossless Priority 3 and 4<br>- 10% - Lossy Background Flow [Priority 0, 1, and 2] <br>3. Enabled PFCWD in default mode before the test. <br>4. Send pause to the egress interface of DUT from IXIA Rx.                                                         | - Single LC - Multiple ASIC <br>- Multiple LC                                    | 1. There should be packet drops on the DUT (IXIA Tx packets > IXIA Rx packets). <br>2. There should PFCs on DUT egress but no PFC sent from DUT ingress to traffic transmitter. <br>3. Only 10% of lossy traffic should flow. <br>4. PFCWD drop counter should increment <br>5. DUT Rx port should be in 'stormed' mode.                                                                                                                                                                       |

```

Comment:
MACsec with PFC is not yet supported in SW/HW on Ixia. Looking for the feasibility of implementing this feature.